lib/app-components/addon/components/project-metadata/component.ts

```diff
@@ -23,28 +22,15 @@ export default class ProjectMetadata extends Component {
     @service toast!: Toast;
 
     node: Node = this.node;
-    makePublicOnSave: boolean = defaultTo(this.makePublicOnSave, false);
+    didValidate: boolean = this.didValidate;
+
     @requiredAction continue!: () => void;
 
     reset = task(function *(this: ProjectMetadata) {
         this.node.rollbackAttributes();
         yield this.node.reload();
     });
 
-    save = task(function *(this: ProjectMetadata) {
-        try {
-            if (this.makePublicOnSave) {
-                this.node.set('public', true);
```

Comment:
The node is not set to public upon submission of the entire form. See [here](https://github.com/CenterForOpenScience/ember-osf-web/pull/568/files#diff-9fee6605a4fe8e48edd73605f2ef014eR84).